packages/backend/src/config/app.test.ts

```diff
@@ -0,0 +1,94 @@
+import * as assert from 'assert'
+import * as crypto from 'crypto'
+import * as fs from 'fs'
+import { parseOrProvisionKey } from './app'
+
+describe('Config', (): void => {
+  describe('parseOrProvisionKey', (): void => {
+    const TMP_DIR = './tmp'
+    const PRIVATE_KEY_FILE = `${TMP_DIR}/private-key.pem`
+
+    beforeEach(async (): Promise<void> => {
+      fs.rmSync(TMP_DIR, { recursive: true, force: true })
+    })
+
+    afterEach(async (): Promise<void> => {
+      fs.rmSync(TMP_DIR, { recursive: true, force: true })
+    })
+
+    test('can provision key', async (): Promise<void> => {
+      expect(fs.existsSync(TMP_DIR)).toBe(false)
+      expect(fs.existsSync(PRIVATE_KEY_FILE)).toBe(false)
+      const { key, jwk } = parseOrProvisionKey(PRIVATE_KEY_FILE)
+      expect(key).toBeInstanceOf(crypto.KeyObject)
+      expect(jwk).toMatchObject({
+        crv: 'Ed25519',
+        kty: 'OKP',
+        d: expect.any(String),
+        x: expect.any(String)
+      })
+      expect(fs.existsSync(PRIVATE_KEY_FILE)).toBe(true)
+    })
+    test('can parse key', async (): Promise<void> => {
+      const keypair = crypto.generateKeyPairSync('ed25519')
+      const keyfile = `${TMP_DIR}/test-private-key.pem`
+      fs.mkdirSync(TMP_DIR)
+      fs.writeFileSync(
+        keyfile,
+        keypair.privateKey.export({ format: 'pem', type: 'pkcs8' })
+      )
+      assert.ok(fs.existsSync(keyfile))
+      const fileStats = fs.statSync(keyfile)
+      const { key, jwk } = parseOrProvisionKey(keyfile)
+      expect(key).toBeInstanceOf(crypto.KeyObject)
+      expect(jwk).toMatchObject({
+        crv: 'Ed25519',
+        kty: 'OKP',
+        d: expect.any(String),
+        x: expect.any(String)
+      })
+      expect(key.export({ format: 'pem', type: 'pkcs8' })).toEqual(
+        keypair.privateKey.export({ format: 'pem', type: 'pkcs8' })
+      )
```

Comment:
https://nodejs.org/api/crypto.html#keyobjectequalsotherkeyobject
```suggestion
      assert.ok(key.equals(keypair.privateKey))
      expect(jwk).toEqual(key.export({ format: 'jwk' }))
      expect(jwk).toEqual({
        crv: 'Ed25519',
        kty: 'OKP',
        d: expect.any(String),
        x: expect.any(String)
      })
```

Reply:
`Property 'equals' does not exist on type 'KeyObject'.`

Reply:
:confused: 
https://nodejs.org/dist/latest-v16.x/docs/api/crypto.html#keyobjectequalsotherkeyobject
Is that a typescript build error?

Reply:
yes
```
src/config/app.test.ts:59:21 - error TS2339: Property 'equals' does not exist on type 'KeyObject'.
59       assert.ok(key.equals(keypair.privateKey))
                       ~~~~~~
Found 1 error.
```
I even made sure to use node v16.18.0.